Changelog

```diff
@@ -24,6 +24,19 @@ Dipy
 
 The code found in Dipy was created by the people found in the AUTHOR file.
 
+* 0.9.0 (Wednesday, 18 March 2015)
+
+* Anatomically Constrained Tissue Classifiers for Tracking
+* Massive speedup (1.5X to 6X depending on data) of Constrained Spherical Deconvolution (CSD)
```

Comment:
No need to remove - just add a comma!


On Wed, Mar 18, 2015 at 12:37 PM, Eleftherios Garyfallidis <
notifications@github.com> wrote:

> In Changelog <https://github.com/nipy/dipy/pull/605#discussion_r26699059>:
>
> > @@ -24,6 +24,19 @@ Dipy
> >
> >  The code found in Dipy was created by the people found in the AUTHOR file.
> >
> > +* 0.9.0 (Wednesday, 18 March 2015)
> > +
> > +* Anatomically Constrained Tissue Classifiers for Tracking
> > +* Massive speedup (1.5X to 6X depending on data) of Constrained Spherical Deconvolution (CSD)
>
> Give me a sec.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/605/files#r26699059>.
>